server/src/main/java/io/spine/server/integration/IntegrationBus.java

```diff
@@ -146,15 +147,15 @@ public void registerWith(BoundedContext context) {
     }
 
     @Override
-    public boolean isRegistered() {
+    public synchronized boolean isRegistered() {
```

Comment:
It seems to be an overkill to make all these methods synchronized.

Let's discuss on a per-case basis.